### ISSUE TITLE
Social share images

### ISSUE THREAD

**Issue by courtneymcgee** [NYPL/staff-picks#141]
Can you add these images for Twitter and Facebook sharing? You will make our social media team SO HAPPY! Be a HERO!

Kids
https://d140u095r09w96.cloudfront.net/sites/default/files/kids-600x335.jpg

Teens
https://d140u095r09w96.cloudfront.net/sites/default/files/teens-600x335.jpg

**Comment by courtneymcgee** [comment]
@wlla slacked me about alt text on the images with a max of 420 characters. Here you go, @EdwinGuzman. 

Kids
https://d140u095r09w96.cloudfront.net/sites/default/files/kids-600x335.jpg
alt = "Featured title: The Book of Mistakes by Corinna Luyken"

Teens
https://d140u095r09w96.cloudfront.net/sites/default/files/teens-600x335.jpg
alt = "Featured title: The Wendy Project by Melissa Jane Osborne, illustrated by Veronica Fish"
